src/lib/menu/menu-foundation.ts

```diff
@@ -16,27 +17,27 @@ export interface IMenuFoundation extends ICustomElementFoundation {
   optionsFactory: MenuOptionFactory | undefined;
   selectedIndex: number;
   selectedValue: any;
-  placement: PopupPlacement;
-  fallbackPlacements: PopupPlacement[];
+  placement: PositionPlacement;
+  fallbackPlacements: PositionPlacement[];
   dense: boolean;
   iconClass: string;
   persistSelection: boolean;
   mode: MenuMode;
-  popupOffset: IPopupPosition;
+  popupOffset: IOverlayOffset;
```

Comment:
Should we alias this as `popoverOffset`? The same goes for any other components with similar properties

Reply:
I think semantically they could still work as "popup" for now, and it will avoid confusion since we plan to deprecate these at some point soon. Eventually with the new patterns in place we will be setting these through the dropdown list element(s) anyway, and in which case I think we'll be using more generic names like just "offset" for those.